CMakeLists.txt

```diff
@@ -19,8 +19,6 @@ include_directories(
   ${urdfdom_headers_INCLUDE_DIRS}
   )
 
-add_compile_options(-std=c++11)
```

Comment:
What do you think about my comment https://github.com/ros-planning/warehouse_ros/pull/92/files#r792535083 ?